<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BucketSortWidget</class>
 <widget class="QWidget" name="BucketSortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/board.png) repeat;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbPeldak">
       <property name="toolTip">
        <string>Rendezendő adatok feltöltése példákkal</string>
       </property>
       <property name="text">
        <string>Példák</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbTorles">
       <property name="toolTip">
        <string>Rendezendő adatok törlése</string>
       </property>
       <property name="text">
        <string>Törlés</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbData">
       <property name="text">
        <string>Rendezendő adatok</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="pteResult"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbResult">
       <property name="text">
        <string>Eredmény</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbTorles</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
